### ISSUE TITLE
Feature request: Shift-Return to create a newline

### ISSUE THREAD

**Issue by jtbx** [squidowl/halloy#202]
Hey, nice client. On some other IRC clients you can hold down Shift and press Return to create a newline in your message. When you press Return to send the message, each line will be individually sent as a new message. I think Halloy should have something like this as it allows a user to send multiple messages at once without being interrupted in a busy channel.

**Comment by casperstorm** [comment]
> i heard this is possible now in iced, right?

Yes! This is possible with https://github.com/iced-rs/iced/pull/2123

**Comment by nc7s** [comment]
It would also be great to be able to Ctrl/Cmd-Enter to send ;)
